src/internal/partial_solution.rs

```diff
@@ -59,6 +59,8 @@ pub struct PartialSolution<P: Package, V: Version> {
     memory: Memory<P, V>,
 }
 
+pub type PackageTerm<P, V> = (P, Term<V>);
```

Comment:
Why is this here instead of in `incompatibility.rs`?

Reply:
No reason besides the return type of functions I was trying to simplify was in `partial_solution.rs`. I will move it to `incompatibility.rs`, it does make more sense there since it's used in `Relation` enum.

Are you fine with it being a type alias rather than a separate type? I have a commit where I had it extracted into a struct, let me know if you would prefer that :smile:

Reply:
I have no methods to add to it so type alias works.

Reply:
Alright. Thank you spotting this! I changed the last commit.